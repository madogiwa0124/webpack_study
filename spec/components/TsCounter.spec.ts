import { mount } from "@vue/test-utils";
import Component from "@js/components/tsCounter.vue";

describe("components/tsCounter.vue", () => {
  describe("初期表示", () => {
    it("snapshot", () => {
      const props = { initCount: 2 };
      const wrapper = mount(Component, { propsData: props });
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
