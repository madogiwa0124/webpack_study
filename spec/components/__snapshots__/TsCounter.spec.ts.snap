// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/tsCounter.vue 初期表示 snapshot 1`] = `
<div
  class="ts-counter"
>
  <button>
    -
  </button>
   2 
  <button>
    +
  </button>
</div>
`;
